<launch>

    <node pkg="yolov4_trt_ros" type="trt_yolo_v4" name="yolov4_trt" output="screen" >
        <param name="topic_name_0" value="/bitcq_camera/image_source1"/>
        <param name="topic_name_1" value="/image_source_1"/>
        <param name="model" value="yolov4"/>
        <param name="model_path" value="$(find yolov4_trt_ros)/yolo/"/>
        <param name="input_shape" type="string" value="416"/>
        <param name="category_number" type="int" value="7"/>
        <param name="confidence_threshold" type="double" value="0.4" />
        <param name="show_image" type="bool" value="true" />
        <param name="nms_iou_threshold" type="double" value="0.5" />
        <param name="match_yaml_path" value="/home/nvidia/ros_ws/src/Image-Matching/config/match_info.yaml"/>
        <param name="bounding_boxes_0_publisher" type="bool" value="true" />
        <param name="bounding_boxes_1_publisher" type="bool" value="true" />
        <param name="bounding_boxes_nms_publisher" type="bool" value="true" />
        <param name="dectec_result_img_publisher" type="bool" value="true" />
    </node>
</launch>
